literature.bib

```diff
@@ -184,6 +184,38 @@ @String{Bachelor
 @String{Project = "Project Thesis"}
 @String{PhD = "PhD Thesis"}
 
+@mastersthesis{Semmler24,
+	author = {Semmler, Sean},
+	title = {{Domain-Specific Variable Ordering for Compiling Binary Decision Diagrams from Feature Models}},
+	school = UniUlm,
+	address = Germany,
+	type = Bachelor,
+	typo3Tags = {SupervisorTH, SupervisorTT},
+	month = JUN,
+	year = 2024,
+	note = {To appear}
+}
+
+@inproceedings{HSST:SPLC24,
+	title = {{Towards Deterministic Compilation of Binary Decision Diagrams From Feature Models}},
+	author = {Heß, Tobias and Semmler, Sean and Sundermann, Chico and Thüm, Thomas},
+	booktitle = SPLC,
+	publisher = ACM,
+	address = NY,
+	note = {To appear},
+	month = SEP,
+	year = 2024
+}
+
+@inproceedings{HOB+:MODEVAR24a,
+	author = {Heß, Tobias and Ostheimer, Lukas and Betz, Tobias and Karrer, Simon and Schmidt, Tim Jannik and Coquet, Pierre and Semmler, Sean and Thüm, Thomas},
+	title = {{variability.dev: Towards an Online Toolbox for Feature Modeling}},
+	booktitle = MODEVAR,
+	note = {{To appear.}},
```

Comment:
Sorry, there is also double brackets and . here.

Reply:
Fixed too.